plugins/RocketsPlugin/jsonSerialization.h

```diff
@@ -267,11 +282,30 @@ inline void init(brayns::Material* m, ObjectHandler* h)
     h->set_flags(Flags::DisallowUnknownKey);
 }
 
+inline void init(brayns::ModelTransformation* g, ObjectHandler* h)
+{
+    h->add_property("translation", Vector3fArray(g->_translation));
+    h->add_property("scale", Vector3fArray(g->_scale));
+    h->add_property("rotation", Vector3fArray(g->_rotation));
+    h->set_flags(Flags::DisallowUnknownKey);
+}
+
+inline void init(brayns::ModelDescriptor* g, ObjectHandler* h)
+{
+    h->add_property("name", &g->_name, Flags::Optional);
+    h->add_property("enabled", &g->_enabled, Flags::Optional);
+    h->add_property("visible", &g->_visible, Flags::Optional);
+    h->add_property("bounding_box", &g->_boundingBox, Flags::Optional);
+    h->add_property("transformations", &g->_transformations);
+    h->add_property("metadata", &g->_metadata);
+    h->set_flags(Flags::DisallowUnknownKey);
+}
+
 inline void init(brayns::Scene* s, ObjectHandler* h)
 {
-    h->add_property("bounds", &s->getWorldBounds(),
-                    Flags::IgnoreRead | Flags::Optional);
-    h->add_property("materials", &s->getMaterials());
+    auto bounds = s->getBounds();
```

Comment:
hmm, the bounds would ideally be cached by the scene, so they don't have to computed here. Let's discuss offline maybe.